src/shared.tsx

```diff
@@ -15,12 +15,6 @@ export const filterOption: SelectProps['filterOption'] = (input, option) => {
   return keyHasAllChars;
 };
 
-export const filterSort: SelectProps['filterSort'] = (a, b) => {
```

Comment:
> it tried to sort by a non-existing key property, instead by name.

I guess the bug is caused by an incompatible change in `antd`. Do the sorting in our provider should ensure the consistent behavior.